MANIFEST.in

```diff
@@ -1,5 +1,5 @@
-include pytorch_lightning/py.typed  # marker file for PEP 561
-include CHANGELOG.md
-recursive-include requirements *.txt
-include requirements.txt
+exclude *.toml  # project config
+exclude requirements.txt
+exclude __pycache__
```

Comment:
This shouldn't be necessary

Reply:
seems it did not do anything in the end :innocent: